README.md

```diff
@@ -61,7 +61,7 @@ Few scenarios are available as example in the [scenarios](https://github.com/Ope
 
 ## Uninstall
 
-To uninstall Open Voice OS run the installer with the `--uninstall` option. even if you dont run the installer with the `--uninstall` option, the installer will check if ovos is installed and will ask you if you want to uninstall it.
+To uninstall Open Voice OS run the installer with the `--uninstall` option. 
```

Comment:
To uninstall Open Voice OS run the installer with the `--uninstall` option *(non-interactive)* or simply run the installer and answer **"Yes"** to the *"Do you want to uninstall Open Voice OS?"* question.

Reply:
thanks :smile: